src/Data/NonEmpty.purs

```diff
@@ -20,9 +20,10 @@ import Data.Eq (class Eq1)
 import Data.Foldable (class Foldable, foldl, foldr, foldMap)
 import Data.FoldableWithIndex (class FoldableWithIndex, foldMapWithIndex, foldlWithIndex, foldrWithIndex)
 import Data.FunctorWithIndex (class FunctorWithIndex, mapWithIndex)
-import Data.Maybe (Maybe(..))
+import Data.Maybe (Maybe(..), maybe)
 import Data.Ord (class Ord1)
 import Data.Semigroup.Foldable (class Foldable1, foldMap1)
+import Data.Semigroup.Foldable (foldl1) as Exports
```

Comment:
I get that you are exporting this so that the `Data.NonEmpty` still exports `foldl1`, but I wonder whether we should keep that since `Foldable1` now includes that as one of its members.